homeassistant/components/sensor/zoneminder.py

```diff
@@ -6,89 +6,82 @@
 """
 import logging
 
-from homeassistant.const import STATE_UNKNOWN
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    STATE_UNKNOWN, CONF_MONITORED_VARIABLES)
 from homeassistant.helpers.entity import Entity
 import homeassistant.components.zoneminder as zoneminder
 
 _LOGGER = logging.getLogger(__name__)
 
 DEPENDENCIES = ['zoneminder']
 
+SENSOR_TYPES = {
+    'events',
+    'function',
+    'status'
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_MONITORED_VARIABLES, default=[]):
```

Comment:
I'm personally fine with making these configurable but the reason I was suggesting showing them all and have users customize to hide is because of backwards compatibility. Perhaps this should default to `['events', 'function']` for backwards compatibility? Unless there is a way to migrate users' `configuration.yaml` upon upgrade?